clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4562,6 +4562,10 @@ renderDebugOptions(const ToolChain &TC, const Driver &D, const llvm::Triple &T,
                              D, TC))
       CmdArgs.push_back("-debug-info-macro");
 
+  // -fno-system-debug turns off debug info generation for system headers
```

Comment:
Is there a driver level test to cover this?

Thanks

Reply:
Driver test added.